calico/getting-started/windows-calico/quickstart.md

```diff
@@ -33,7 +34,7 @@ This is required to prevent Linux nodes from borrowing IP addresses from Windows
 calicoctl ipam configure --strictaffinity=true
 ```
 
-#### Install {{site.prodnameWindows}}
+#### Install {{site.prodnameWindows}} manually
 
 The following steps install a Kubernetes cluster on a single Windows node, with a Linux control node.
```

Comment:
Ditch the comma... The following steps install a Kubernetes cluster on a single Windows node with a Linux control node.